src/styles/annotator/highlights.scss

```diff
@@ -124,6 +127,7 @@
 // Apply styling using `--highlight-` values when highlights are visible
 // The `.hypothesis-highlights-always-on` class is managed by `highlighter`
 .hypothesis-highlights-always-on .hypothesis-svg-highlight {
+  transition: fill 300ms;
```

Comment:
I think this looks kind of nice when changing cluster colors, but it's discretionary...

Reply:
I don't mind the gradual animation when changing cluster colors, as that's a global change, but this also ends up affecting hover states and I think that the transition there (if any) should be either immediate or at least much faster. In the sidebar we typically use durations of 150ms (with a couple of exceptions).

Reply:
I think I'll pull it for now, as it's fluff/tuning anyway.